repose-aggregator/tests/functional-test-framework/build.gradle

```diff
@@ -21,6 +21,12 @@ processResources {
     )
 }
 
+processScoverageResources {
```

Comment:
I really hate how scoverage works. Compile time instrumentation is terrible.

Reply:
Especially given the high cost of scala compilation

Reply:
This was the entire issue yesterday. It wasn't that it couldn't find the file, it was that the `version` variable hadn't been expanded when it copied it.